api/rbdigital.py

```diff
@@ -142,6 +142,24 @@ class RBDigitalAPI(BaseCirculationAPI, HasSelfTests):
     # a complete response returns the json structure with more data fields than a basic response does
     RESPONSE_VERBOSITY = {0:'basic', 1:'compact', 2:'complete', 3:'extended', 4:'hypermedia'}
 
+    CACHED_IDENTIFIER_PROPERTY = 'patronId'
+    BEARER_TOKEN_PROPERTY = 'bearer'
+
+    # Parameterize credentials.
+    # - The `label` property maps to Credential `type`.
+    # - The `lifetime` is used to calculate Credential `expires`. If it
+    #   is None, then the Credential does not expire.
+    CREDENTIAL_TYPES = {
+        CACHED_IDENTIFIER_PROPERTY: dict(
+            label=Credential.IDENTIFIER_FROM_REMOTE_SERVICE,
+            lifetime=None
+        ),
+        BEARER_TOKEN_PROPERTY: dict(
+            label="Patron Bearer Token",
+            lifetime=(23 * 60 + 30) * 60),
```

Comment:
I figured out that this is 23 hours and 30 minutes but it was hard to read. Can you represent this either as `(24*60*60) * 0.9` or `((24*60) - 30) * 60`, and add a comment saying that it's slightly less than the lifetime RBDigital advertises?